.travis.yml

```diff
@@ -17,7 +17,8 @@ before_install:
     - travis_retry conda create --yes -n test python=$PYTHON --file requirements.txt
     - source activate test
     - conda install --yes --file requirements-dev.txt
-    - travis_retry conda install --yes pytest pandas vincent flake8
+    - travis_retry conda install --yes pytest pandas vincent flake8 nbconvert jupyter_client ipykernel shapely fiona
```

Comment:
Provided they are in `requirements-dev.txt`, can we remove this line ?
(I don't know what `travis_retry` does).